net.studio08.xbplay.metainfo.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id type="desktop">net.studio08.xbplay</id>
+    <launchable type="desktop-id">net.studio08.xbplay.desktop</launchable>
+    <name>xbPlay: Remote Play for Xbox</name>
+    <summary>Remote play from your console or xCloud</summary>
+    <metadata_license>CC-BY-SA-4.0</metadata_license>
+    <project_license>Apache-2.0</project_license>
+    <developer_name>Studio08 Development</developer_name>
+    <url type="homepage">https://www.reddit.com/r/xbPlay/</url>
+    <description>
+        <p>
+            Remote play from your Xbox One or Series X/S game console directly to your Linux device.<br></br>
+            It supports 1080p quality, backward compatible games, and xCloud titles. Haptic feedback is also supported on connected controllers. The xbPlay app is available on Google Play, Google Play TV, the iOS App Store, and the Amazon Marketplace (TV).<br></br>
+            *This is the linux port of the xbPlay app.
```

Comment:
Not sure there's any point in mentioning it's a Linux port.

Reply:
Ok removed 👍